### PR TITLE
Notification view

### LINKED ISSUE #54: [2pts] Notifications Screen
**Fill in the blanks**:
As a student, I would like to view enableable notifications, so that I can get notifications for things I want.

**Describe the solution you'd like:**
- [ ] Add a title to identify the page
- [ ] Create a toggle for 'favorite foods' to add functionality to later
- [ ] Have a 'back' button to add functionality to later
- [ ] Contains Nav Bar to add functionality to later

**Definition of Done - Check as you go!**
- [ ] Tasks have been implemented
- [ ] Documented
- [ ] Passing Tests for each task
- [ ] Pull Request made
- [ ] Code Reviewed

### PR DESCRIPTION
**Pull request by DDVD233**
I largely used the approach @RafaelPiloto10 used on the home screen.

### CODE REVIEW

**Comment by DDVD233** [comment]
Maybe I should wait for #56 to be merged

**Comment by RafaelPiloto10** [comment]
Can you please attach screenshots @DDavid233?

**Comment by DDVD233** [comment]
image
Notifications page.
image
Deletion implemented

File: quack_app/test/core/notification_data_test.dart
Comment (RafaelPiloto10) [inline quack_app/test/core/notification_data_test.dart]: Please change the group name

File: quack_app/test/core/notification_data_test.dart
Comment (RafaelPiloto10) [inline quack_app/test/core/notification_data_test.dart]: This will mean that the mocked notifications class in the future will need to return 3. Consider testing that the notifications list is not empty rather than a specific length

File: quack_app/test/core/notification_data_test.dart
Comment (RafaelPiloto10) [inline quack_app/test/core/notification_data_test.dart]: Same here - rather than hard coding the length, get the length of total notifications and expect `length - 1` rather than hard coding `2`

File: quack_app/test/screens/notification/notification_screen_test.dart
Comment (RafaelPiloto10) [inline quack_app/test/screens/notification/notification_screen_test.dart]: You may want to take the same approach here. Find the length of total notifications first before deletion, then expect `length - 2` instead of hard coding it here.

**Review by RafaelPiloto10 — CHANGES_REQUESTED** [review]
Just some minor testing convenience things. Everything else looks great! Nice job! 

Before you merge this request, you might want to take a stab at a reactive notifications icon in the navbar. If the user has any notifications, display the icon with the red circle bell indicating that the user has pending notifications. 

If it looks like it will take more than 10 mins to add, don't worry about it! We can make a ticket for it later. All in all, looks great :)

**Review by RafaelPiloto10 — APPROVED** [review]

**Review by brendacano — APPROVED** [review]
It all looks good to me

**Pull request merged**
